homeassistant/components/frontend/__init__.py

```diff
@@ -497,23 +486,17 @@ def get(self, request, extra=None):
             # do not try to auto connect on load
             no_auth = 'false'
 
-        template = yield from hass.async_add_job(
-            self.templates.get_template, 'index.html')
+        template = yield from hass.async_add_job(self.get_template, latest)
 
-        # pylint is wrong
-        # pylint: disable=no-member
-        # This is a jinja2 template, not a HA template so we call 'render'.
         resp = template.render(
-            core_url=core_url, ui_url=ui_url,
-            compatibility_url=compatibility_url, no_auth=no_auth,
-            icons_url=icons_url, icons=icons_fp,
-            panel_url=panel_url, panels=hass.data[DATA_PANELS],
-            dev_mode=self.use_repo,
+            no_auth=no_auth,
+            panel_url=panel_url,
+            panels=hass.data[DATA_PANELS],
+            dev_mode=self.repo_path is not None,
             theme_color=MANIFEST_JSON['theme_color'],
             extra_urls=hass.data[DATA_EXTRA_HTML_URL],
             latest=latest,
-            service_worker_name='/service_worker.js' if latest else
-            '/service_worker_es5.js')
+        )
```

Comment:
Should add back service_worker_url

Reply:
No, it's not needed because I hardcode `/service_workerj.js` in the index.html template. It's [replaced](https://github.com/home-assistant/home-assistant-polymer/pull/609/files#diff-acc83fcf726d32b30ae0e1c1c06d0f4cR24) by `/service_worker_es5.js` for the ES5 version.